src/main/java/uk/gov/digital/ho/hocs/casework/migration/api/dto/CaseAttachment.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.digital.ho.hocs.casework.migration.api.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+@Getter
+public class CaseAttachment {
+
+    @JsonProperty("displayName")
+    private String displayName;
+
+    @JsonProperty("documentType")
+    private String type;
+
+    @JsonProperty("s3UntrustedUrl")
```

Comment:
As commented on the other one - I think this probably should be `documentUuid` or similar

Reply:
> As commented on the other one - I think this probably should be `documentUuid` or similar

I don't think this is the document UUID though? It's the URL of the file in the untrusted bucket which happens to be a UUID?

Reply:
`documentPath` then? Or even `fileLink` as you suggested? I don't see the value of having `s3Untrusted` nor `Url` as its not strictly accurate

Reply:
Updated.